lib/nacs-seq/manager.cpp

```diff
@@ -391,6 +391,26 @@ void Manager::unref_data(uint64_t id)
     m_datainfo.erase(it);
 }
 
+NACS_EXPORT() uint32_t Manager::get_device_restart(const char *dname)
+{
+    // This function should not throw. Returns uint32_t(-1) if device not found.
+    auto it = m_device_info.find(dname);
+    if (it != m_device_info.end())
+        return it->second.n_restarts;
+    return uint32_t(-1);
+}
+
+NACS_EXPORT() bool Manager::add_device_restart(const std::string &name, uint32_t n = 1)
```

Comment:
Missed it last time. Default value belongs to the declaration.

Reply:
Oh okay, didn't know that! One last amend coming...